apps/charterafrica/src/components/EmbeddedDocumentViewer/useSourceAfrica.js

```diff
@@ -0,0 +1,17 @@
+import useSWR from "swr";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+function useSourceAfrica(url, args) {
```

Comment:
```suggestion
function useDocument(url, options) {
```

We need documents, source.AFRICA is the whole app.